qa-ibd.planx-pla.net/manifest.json

```diff
@@ -4,24 +4,24 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2020.03",
+    "arborist": "quay.io/cdis/arborist:2020.05",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "dashboard": "quay.io/cdis/gen3-statics:1.0.0",
-    "fence": "quay.io/cdis/fence:2020.03",
-    "indexd": "quay.io/cdis/indexd:2020.03",
-    "peregrine": "quay.io/cdis/peregrine:2020.03",
-    "pidgin": "quay.io/cdis/pidgin:2020.03",
+    "fence": "quay.io/cdis/fence:2020.05",
+    "indexd": "quay.io/cdis/indexd:2020.05",
+    "peregrine": "quay.io/cdis/peregrine:2020.05",
+    "pidgin": "quay.io/cdis/pidgin:2020.05",
     "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
-    "sheepdog": "quay.io/cdis/sheepdog:2020.03",
-    "portal": "quay.io/cdis/data-portal:2.24.8",
+    "sheepdog": "quay.io/cdis/sheepdog:2020.05",
+    "portal": "quay.io/cdis/data-portal:2020.05",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "jupyterhub": "quay.io/occ_data/jupyterhub:master",
-    "spark": "quay.io/cdis/gen3-spark:2020.03",
-    "tube": "quay.io/cdis/tube:0.3.26",
-    "hatchery": "quay.io/cdis/hatchery:2020.03",
-    "wts": "quay.io/cdis/workspace-token-service:2020.03",
+    "spark": "quay.io/cdis/gen3-spark:2020.05",
+    "tube": "quay.io/cdis/tube:2020.05",
+    "hatchery": "quay.io/cdis/hatchery:2020.05",
```

Comment:
we can add a hatchery block here similar to prod https://github.com/uc-cdis/cdis-manifest/blob/master/ibdgc.datacommons.io/manifest.json#L45